docs/methoddocs/filleeventqueries.md

```diff
@@ -1 +1,300 @@
 # File Event Queries
+
+```eval_rst
+.. autoclass:: py42.clients.file_event.FileEventClient
+    :members:
+    :show-inheritance:
+```
+
+```eval_rst
+.. autoclass:: py42.sdk.queries.fileevents.file_event_query.FileEventQuery
+    :members:
+    :show-inheritance:
+```
+
+# Filter Classes
+
+Each of the following classes can be used to filter Forensic Search results based on the class's corresponding event
```

Comment:
I try to avoid writing sentences like "can be used by" because Grammarly always yells at me. My suggestion is something like "To filter Forensice Search results, use any of the following classes..."

`To `

Reply:
Reworked the paragraph. Nice and crisp now.